### PR TITLE
[PLA-13576] Add model assistant support.

### PR DESCRIPTION
**Pull request by anoto-moniz**
# Citrine Python PR

## Description 
Please briefly explain the goal of the changes/this PR.
The reviewer should be able to understand why the change is being made by reading this description
and its links (e.g. JIRA tickets).

### PR Type:
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Maintenance (non-breaking change to assist developers)

### Adherence to team decisions
- [ ] I have added tests for 100% coverage
- [ ] I have written Numpy-style docstrings for every method and class.
- [ ] I have communicated the downstream consequences of the PR to others.
- [ ] I have bumped the version in __version\__.py

### CODE REVIEW

File: src/citrine/resources/response.py [hunk 1]
```diff
@@ -22,7 +22,7 @@ def _get_body_string(self):
             return "No body available"
 
     def __repr__(self):
-        return 'Response({!r}), {!r})'.format(self._get_status_string(), self._get_body_string())
+        return 'Response({!r}, {!r})'.format(self._get_status_string(), self._get_body_string())
```
Comment (kroenlein) [inline src/citrine/resources/response.py:25]: As long as you're editing, why not migrate to an f-string?

**Review by kroenlein — APPROVED** [review]
Good abstraction for the URL escaping.  I can't speak to the Catalyst API implementation.

File: src/citrine/resources/response.py (hunk 1 above)
Comment (anoto-moniz) [reply src/citrine/resources/response.py:25]: Good call! This was in my `git stash` from a ways back, so I didn't even think to do that.

**Review by kroenlein — APPROVED** [review]
LGTM

**Pull request merged**